### PR TITLE
CHANGELOG for 0.9.0

### PR DESCRIPTION
**Pull request by angelozerr**
CHANGELOG for 0.9.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by jeffmaury** [comment]
There always have been an issue with version identifiers: current version is 0.9.0-SNAPSHOT so I think this one should be updated accordingly

**Comment by angelozerr** [comment]
> There always have been an issue with version identifiers: current version is 0.9.0-SNAPSHOT so I think this one should be updated accordingly

Good catch @jeffmaury ! Fixed, thanks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# Quarkus extension for MicroProfile Language Server Changelog
+
+## 0.9.0 (September 21, 2020)
+
+Since `0.9.0`, quarkus-ls becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), see [CHANGELOG of vscode-lsp4mp](https://github.com/eclipse/lsp4mp/CHANGELOG.md#010-september-14-2020)
```
Comment (fbricon) [inline CHANGELOG.md:5]: becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), the Language Server for Eclipse MicroProfile, see the upstream [lsp4mp CHANGELOG] for additional informations about enhancements and bug fixes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+### Build
+
+ * quarkus.jdt.ext should refer to quarkus-jdt sub folder. See [#353](https://github.com/redhat-developer/quarkus-ls/issues/353).
+
+### Other
+
+ * Remove jax-rs snippets. See [#362](https://github.com/redhat-developer/quarkus-ls/pull/362).
+ * Adapt README with LSP4MP. See [#352](https://github.com/redhat-developer/quarkus-ls/issues/352).
+ * Rename `quarkus` prefix settings (used by language server) with `microprofile` . See [#325](https://github.com/redhat-developer/quarkus-ls/issues/325).
+ * Create a new project com.redhat.quarkus.ls. See [#295](https://github.com/redhat-developer/quarkus-ls/issues/295).
```
Comment (fbricon) [inline CHANGELOG.md:21]: Build category

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+ * Split Quarkus / MicroProfile LS support. See [#268](https://github.com/redhat-developer/quarkus-ls/issues/268).
```
Comment (fbricon) [inline CHANGELOG.md:22]: Build category

File: CHANGELOG.md (hunk 1 above)
Comment (fbricon) [reply CHANGELOG.md:5]: should link to the sept 21 anchor of lsp4mp's changelog

**Review by fbricon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
+# Quarkus extension for MicroProfile Language Server Changelog
+
+## 0.9.0 (September 21, 2020)
+
+Since `0.9.0`, quarkus-ls becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), the Language Server for Eclipse MicroProfile, see the upstream [lsp4mp CHANGELOG](https://github.com/eclipse/lsp4mp/CHANGELOG.md#010-september-21-2020)
```
Comment (fbricon) [inline CHANGELOG.md:5]: add missing period at the end of the sentence.

Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
+# Quarkus extension for MicroProfile Language Server Changelog
+
+## 0.9.0 (September 21, 2020)
+
+Since `0.9.0`, quarkus-ls becomes an extension of [lsp4mp](https://github.com/eclipse/lsp4mp), the Language Server for Eclipse MicroProfile, see the upstream [lsp4mp CHANGELOG](https://github.com/eclipse/lsp4mp/CHANGELOG.md#010-september-21-2020).
```
Comment (fbricon) [inline CHANGELOG.md:5]: changelog link doesn't work. Try https://github.com/eclipse/lsp4mp/blob/master/CHANGELOG.md#010-september-21-2020

Comment (angelozerr) [reply CHANGELOG.md]: Good catch! Fixed. Please merge my PR for vscode-quarkus https://github.com/redhat-developer/vscode-quarkus/pull/305 which have the same problem.

**Pull request merged**